lib/perl/Genome/Model/ReferenceAlignment/Command/GenotypeMicroarrayConcordance.pm

```diff
@@ -38,6 +39,19 @@ class Genome::Model::ReferenceAlignment::Command::GenotypeMicroarrayConcordance
         },
     ],
     has_optional =>[
+        microarray_build => {
+            is => 'Genome::Model::Build::GenotypeMicroarray',
+            doc => 'Use this microarray build as the truth VCF regardless of default genotype data for the sample.',
+        },
+        lane_qc_builds => {
```

Comment:
This might be better named 'lookup_lane_qc_builds' to indicate that it is not itself a list of the builds but is a boolean option.